liquidctl-completion.sh

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/env bash
+
+# This is a bash completion script for liquidctl 
+# to enable this completion place this file in /etc/bash_completion.d/ (note this locatio may be system dependant_ 
+# you can also enable this manually by sourcing this file in your ~/.bashrc file `source path/to/this` 
+#
+# Created by: Marshall Asch
```

Comment:
Unless this is super common convention for completion files, I think it's better to have an explicit copyright and license notice.

```
Copyright (C) 2020–2020  Marshall Asch
SPDX-License-Identifier: GPL-3.0-or-later
```

Reply:
nope there isnt a convention, I'm just new to "good" file headers, most of the files I have put it on have been for classes.

Reply:
The format you arrived at was fine in isolation, but since we're shipping a bunch of stuff together, I think it's easier to just normalize credits/copyright/license info in a single format.